src/celengine/star.h

```diff
@@ -241,8 +241,12 @@ class Star : public AstroObject
     Star() = default;
     virtual ~Star();
 
+    Star(Star &&s);
+
```

Comment:
add `Star(const Star&)` and appropriate operators `operator=`

Reply:
Should I? AstroObject doesn't have copy constructor either. I think objects shouldn't be copy-able.

Reply:
in this case delete the explicitly:
```
Star(const Star&) = delete;
```
I'm sorry but C++ requires this.

Reply:
Ok. But why not to delete it in `AstroObject` only? Should I delete it in every derived class?

Reply:
derived classes should derive all constructors, and deleted too